psi4/src/psi4/lib3index/dfhelper.h

```diff
@@ -52,6 +52,8 @@ class TwoBodyAOInt;
 class PSI_API DFHelper {
    public:
     DFHelper(std::shared_ptr<BasisSet> primary, std::shared_ptr<BasisSet> aux);
+    DFHelper(std::shared_ptr<BasisSet> primary, std::shared_ptr<BasisSet> aux,
+        Options& options);
```

Comment:
There's no easy way around adding Options to DFHelper?

Reply:
Theoretically, one could simply pass the setting of the "SCF_SUBTYPE" keyword itself through the constructor instead of the entire options object, as "SCF_SUBTYPE" is the only option DFHelper really uses right now. I went with this way, though, because I consider it more elegant (more in line with the JK constructors), and it helps future-proof in the case that options are needed in DFHelper in the future.

Reply:
Looking into it some more, it may be possible to access the "SCF_SUBTYPE" variable through Process::environment.globals instead of passing an options object. I'll be happy to switch to this methodology if that is preferred!

Reply:
So it seems Process::environment can be used to get "SCF_SUBTYPE", instead of passing an options object! If this is the preferred methodology, I'll be happy to use that way instead.

Reply:
After further discussion with Lori, I have reimplemented options access in DFHelper using Process::environment!